cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/StreamHandler.java

```diff
@@ -478,7 +478,7 @@ public JsonElement serialize(StreamProperties src, Type typeOfSrc, JsonSerializa
       json.addProperty("name", src.getName());
       json.addProperty("ttl", TimeUnit.MILLISECONDS.toSeconds(src.getTTL()));
       json.add("format", context.serialize(src.getFormat(), FormatSpecification.class));
-      json.addProperty("threshold", src.getThreshold());
+      json.addProperty("notification_threshold_mb", src.getNotificationThresholdMB());
```

Comment:
only having serializer but not deserializer seems wrong to me. Can you check the existing logic and fix that as well?